tests/test_connector.py

```diff
@@ -3307,10 +3307,7 @@ async def handler(request: web.Request) -> NoReturn:
 
     srv = await aiohttp_server(app, ssl=ssl_ctx)
 
-    port = unused_port()
-    conn = aiohttp.TCPConnector(local_addr=("127.0.0.1", port))
```

Comment:
This one appears to always have been failing but now the exception about the duplicate port use is obvious

Reply:
nope, we needed to trap connector errors and prevent retry. all fixed